promscale/visualize-data/index.md

```diff
@@ -0,0 +1,11 @@
+# Visualize data in Promscale
+Promscale can be used to visualize the data in tools like
```

Comment:
I think this sentence is written backwards. We should say that different tools can be used to visualize data in Promscale and not the other way around.

Reply:
```suggestion
You can visualize data in Promscale using tools like
```

Is that better?